pkg/hive/resources.go

```diff
@@ -10,16 +10,18 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Changing values of these constants most likely would require
+// some sort of migration on the Hive cluster for existing clusters.
 const (
-	ClusterDeploymentName      = "cluster"
-	kubesecretName             = "admin-kube-secret"
-	servicePrincipalSecretname = "serviceprincipal-secret"
+	ClusterDeploymentName             = "cluster"
+	aroServiceKubeconfigSecretName    = "aro-service-kubeconfig-secret"
+	clusterServicePrincipalSecretName = "cluster-service-principal-secret"
```

Comment:
We should not be changing these values (added a comment about it), but since we are still in development - it is ok.